aepsych/database/db.py

```diff
@@ -58,6 +58,8 @@ def get_engine(self):
     def delete_db(self):
         if self._engine is not None and self._full_db_path.exists():
             os.remove(self._full_db_path.as_posix())
+            close_all_sessions()
```

Comment:
@gstocek wondering if you can help us out here -- this is a pretty heavy-handed fix (I believe it'll close unrelated sessions that are started within the same python process), but @AxlCruzTKH had trouble getting a narrower version to work. An alternate workaround is to remove this method entirely, since it's only used in testing, and instead have something like this in a testing `tearDown` method, at which point I'm a bit more comfortable in closing all sessions.